benchmarking/data/simulated_data/mock_student_provider.py

```diff
@@ -241,3 +266,30 @@ def attribute_values_from_range(
         replace=False,
         generator=_generator,
     )
+
+
+def _generate_attribute_values_without_probability(
+    attribute_ranges: Dict[int, AttributeRangeConfig], class_size: int
+):
+    probabilistic_attributes = {}
+    for attribute_id, attribute_range_config in attribute_ranges.items():
+        if isinstance(attribute_range_config[0], (int, AttributeValueEnum)):
```

Comment:
Wait so what happens if this is the case then? Like the dictionary would just have nothing in it for these attributes